### PR TITLE
Changes due to re-factoring for share extension

### PR DESCRIPTION
**Pull request by typfel**
This PR adapts to interface changes made in data model and transport session for supporting share extensions.

depends on:
- [ ] https://github.com/wireapp/wire-ios-data-model/pull/83
- [ ] https://github.com/wireapp/wire-ios-transport/pull/21
- [ ] https://github.com/wireapp/wire-ios-message-strategy/pull/21

### CODE REVIEW

**Comment by daehn** [comment]
💣 Conflicts

**Comment by typfel** [comment]
❗️Note: The hotfix number needs to be bumped to the current version.

**Comment by mikeger** [comment]
@typfel you mean to bump all the dependencies of the sync engine?

File: Source/Synchronization/ZMHotFixDirectory.m [hunk 1]
```diff
@@ -85,6 +85,11 @@ - (NSArray *)patches
                      patchCode:^(NSManagedObjectContext *context) {
                          [ZMHotFixDirectory removeDeliveryReceiptsForDeletedMessages:context];
                      }],
+                    [ZMHotFixPatch
+                     patchWithVersion:@"60.0.0"
```
Comment (typfel) [inline Source/Synchronization/ZMHotFixDirectory.m]: This SE is already released so we need to bump it.

**Comment by mikeger** [comment]
@typfel bumped to 61.0.0. Thanks!

**Pull request merged**